packages/cli/src/utils/directory.ts

```diff
@@ -21,10 +21,18 @@ export const srcFolderName = 'src'
 export const userSrcDir = path.join(userDir, srcFolderName)
 
 export const customizationsFolderName = 'customizations'
```

Comment:
Maybe we can just ignore this file change 🙈  Just indentation 🤔

Reply:
My bad 🤡